IDE/VisualStudioForMac/Properties/AddinInfo.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using Mono.Addins;
 using Mono.Addins.Description;
-[assembly: Addin("MonoGame_IDE_VisualStudioForMac",Namespace = "MonoDevelop",Version = "0.0.0.0")]
+[assembly: Addin("MonoGame_IDE_VisualStudioForMac",Namespace = "MonoDevelop",Version = "3.7.0.0")]
```

Comment:
This number is getting set by the build system and should not be modified.

Reply:
ok, the visual studio code change automaticly when i opened the project. I'm gonna ignore next time